packages/smoke-test/test/commands/__snapshots__/fix.test.ts.snap

```diff
@@ -679,7 +683,8 @@ import GreetingHeader from \\"./GreetingHeader\\";
  * @param {string} name the name to format
  * @returns {string} the formatted name
  */
-function formatName(name: string): string {
+// @ts-expect-error @rehearsal TODO TS80004: JSDoc types may be moved to TypeScript types.
+function formatName(name: string) {
```

Comment:
looks like we've lost the return type here. should be `function formatName(name: string): string`

Reply:
This has been addressed.